res/css/style.css

```diff
@@ -507,7 +507,7 @@ body, #root, .profileViewer {
 
 .react-contextmenu.react-contextmenu--visible {
   display: block;
-  max-height: 43.5vh;
+  max-height: 70vh;
```

Comment:
Hi @julienw, yes I've tested with values higher than 70vh. I've considered the zoom of up to 150%, with a higher zoom the bottom half of the screen is too small to be really effective and a very large context menu only makes things more difficult to navigate. I would suggest increasing the size of the call tree when the zoom is higher than 150%. If we were to consider zoom of up to 200% - which I think is recommended.

Reply:
I've added a brief comment to the code as well :)

Reply:
Wow, I've never done an amend before, I hope this works (thank you for sharing!:)

Reply:
I still think we should put something more, like 90vh :) 100vh is too much as this would hide the fact it's a menu, but I think 90vh would be good.

As an example, you can try https://deploy-preview-1328--perf-html.netlify.com/public/dfc1b84f9982b4d67e0c1079bad2c582cde1f32f/calltree/?globalTrackOrder=0-1-2-3&hiddenGlobalTracks=1-2-3&hiddenLocalTracksByPid=2399-0-1&localTrackOrderByPid=2399-0-1~2422-0~&thread=0&v=3
(notice I use the deploy preview for this PR, so this has your changes)

In this example, there's only 1 thread displayed. I zoomed until 300% and I think it's still usable, at least with my screen resolution (1920x1200). But when I right click something in call tree, I get a scrollbar, while we could avoid it because there's still some vertical space.

See the screenshot below:
![capture d ecran de 2018-10-09 15-08-05](https://user-images.githubusercontent.com/454175/46671475-3146db80-cbd5-11e8-884d-66127b4274e7.png)